src/gui/src/mainWindow.cpp

```diff
@@ -391,6 +402,17 @@ const std::string MainWindow::addToolbarButton(const std::string& name,
   }
 
   auto action = view_tool_bar_->addAction(text);
+  // save the command so it can be restored later
+  QString escaped_script = script;
+  QString cmd = "gui::create_toolbar_button ";
+  cmd += "{" + QString::fromStdString(name) + "} {" + text + "} ";
+  cmd += "{" + escaped_script + "} ";
```

Comment:
replace with script and delete escaped_script since it isn't escaped anymore or use anywhere else.